src/main/java/org/heigit/ohsome/ohsomeapi/output/metadataresponse/MetadataResponse.java

```diff
@@ -19,10 +19,12 @@
 
   @ApiModelProperty(notes = "License and copyright info", required = true, position = 0)
   private Attribution attribution;
-  @ApiModelProperty(notes = "Version of this api", required = true, position = 0)
+  @ApiModelProperty(notes = "Version of this api", required = true, position = 1)
```

Comment:
puh, perhaps this could be slightly confusing, because as a user I would expect `v1` to be the “_Version of this API_”…

```suggestion
  @ApiModelProperty(notes = "Software version running this API", required = true, position = 1)
```

Reply:
Hmmm.. these docs here are quite old actually and are only shown in the swagger JSON file.
To me "v1" and "v1.2.3" are both expressions of the version of the API. The respective JSON field in every response is also just called "apiVersion".

Also the wording would be different, cause the API runs on a specific version, not the other way around.
What about `Specific version on which this API runs`? Using "specific" here in reference to the major.minor.patch pattern.

Reply:
> To me "v1" and "v1.2.3" are both expressions of the version of the API.

Well, strictly speaking it would be `v1` versus `1.2.3` (sans `v`). But I get the point that they are not independent.

> What about `Specific version on which this API runs`? Using "specific" here in reference to the major.minor.patch pattern.

This should work as well. Maybe it could even be shortened a bit:

```suggestion
  @ApiModelProperty(notes = "Specific version of this API", required = true, position = 1)
```